tests/vmcheck/test-kernel-args.sh

```diff
@@ -180,6 +180,22 @@ vm_rpmostree kargs --delete-if-present=PACKAGE3=TEST3 --unchanged-exit-77 || rc=
 assert_streq $rc 77
 echo "ok exit 77 when unchanged kargs with unchanged-exit-77"
 
+# Test for 'rpm-ostree kargs --editor'.
+vm_rpmostree kargs > kargs.txt
+assert_not_file_has_content_literal kargs.txt 'nonexisting'
+rc=0
+EDITOR="sed -i 's/nonexisting//'" vm_rpmostree kargs --editor --unchanged-exit-77 || rc=?
+assert_streq $rc 77
+vm_rpmostree kargs > kargs.txt
+assert_not_file_has_content_literal kargs.txt 'nonexisting'
+vm_rpmostree kargs > kargs.txt
+assert_not_file_has_content_literal kargs.txt 'editorkey=editorvalue'
+rc=0
+EDITOR="sed -i 's/$/ editorkey=editorvalue/'" vm_rpmostree kargs --editor --unchanged-exit-77
+assert_streq $rc 0
```

Comment:
This isn't actually capturing the return code. I think for invocations we to succeed that makes sense, so we can just delete this line and the `rc=0` line. I'm guessing this is based on the previous tests; I think we should do that cleanup there too for the invocations which should succeed, but feel free to leave as a follow-up.